spec/models/replication/form_replication_spec.rb

```diff
@@ -131,6 +131,32 @@
       end
     end
 
+    context "with a question with multiple conditions" do
```

Comment:
using the keys means this will hold up with switch to uuids, right?

Reply:
Yes.